pkg/docker/docker.go

```diff
@@ -308,7 +308,7 @@ func New(config *api.DockerConfig, auth api.AuthConfig) (Docker, error) {
 	if err != nil {
 		return nil, err
 	}
-	k8sDocker := dockertools.ConnectToDockerOrDie(config.Endpoint, 0)
```

Comment:
yeah i think we need to solve why the default logic isn't working, not remove it entirely.  but @jim-minter fyi.